src/main/java/uk/gov/hmcts/reform/sendletter/model/Letter.java

```diff
@@ -22,11 +23,16 @@
     @NotEmpty
     public final String type;
 
+    @JsonProperty("additional_data")
```

Comment:
Not needed as defined in constructor. Maybe provide `ApiModelProperty` instead?

Reply:
this is for Jackson for serialization

Reply:
so other properties also need then, right?

Reply:
only if it's different than field name (here it's 2 words and guideline says snake_case should be used instead of camelCase